### ISSUE TITLE
Shapely runtime warnings in newer climada_env builds

### ISSUE THREAD

**Issue by sarah-hlsn** [CLIMADA-project/climada_python#864]
Newer versions of the climada_env build (not sure exactly when this issue started) are getting many shapely runtime warnings (mostly of the kind: invalid value in buffer/intersection). This issue seems to come from somewhere outside climada, but it does not seem to be in relationship to the version of shapely installed.

**Comment by emanuel-schmid** [comment]
It's possibly related to the OS though - haven't encountered it neither on Linux nor Windows.
(Please report if you prove me wrong)

**Comment by peanutfun** [comment]
I also get flooded with shapely warnings due to calls to the `buffer` function somewhere in `Hazard.plot_intensity`, but I could not track the source so far.

**Comment by peanutfun** [comment]
@sarah-hlsn Still relevant?

**Comment by sarah-hlsn** [comment]
@peanutfun No, the shapely warnings no longer show up! I'm using 6.0.2.dev0 but I imagine they wouldn't show up after climada release 6.0.0 :)
